.github/workflows/zot-minimal-size.yml

```diff
@@ -0,0 +1,40 @@
+name: "zot minimal binary size"
+on:
+  pull_request:
+    branches: [main]
+
+permissions: read-all
+
+jobs:
+  zot-minimal-size:
```

Comment:
Why not just make this a step in ci-cd.yml itself?

Reply:
I like this check btw.

Reply:
> Why not just make this a step in ci-cd.yml itself?

It was a step in ci-cd.yml initialy. Team decision was to add a new workflow instead for two reasons:
1. Fail faster in case zot minimal binary increases in PR instead of waiting other steps from ci-cd.yml to complete
2. Not add an extra 2 min in ci-cd workflow and wait longer for it to finish

Reply:
An additional argument to keep it separate.
3. If it fails in ci-cd.yaml the `build-test / Build and test ZOT (linux, amd64) (pull_request)` GH check would be red, Having a separate check makes it more easy for reviewer to assess the state of the PR.